### PR TITLE
Fix minor typo in the availability-zone help command

### PR DESCRIPTION
**Pull request by Holmistr**

### CODE REVIEW

File: cmd/availabilityZone.go
Comment (EugeneAbramchuk) [inline cmd/availabilityZone.go:26]: I'd rephrase this as well to something like:
Availability zones depend on the cloud provider, region, instance type and count

(currently it's less clear and has minor typos)

Comment (abramche) [inline cmd/availabilityZone.go]: I'd rephrase this as well to something like:
Availability zones depend on the cloud provider, region, instance type and count

(currently it's less clear and has minor typos)

Comment (Holmistr) [reply cmd/availabilityZone.go]: Good point, let me update.

**Review by abramche — APPROVED** [review]

**Review by abramche — APPROVED** [review]

**Review by huseyinbabal — APPROVED** [review]

**Comment by huseyinbabal** [comment]
Please inform once you are done with the changes, it can be easily released after merge

**Review by yunussandikci — APPROVED** [review]

**Comment by yunussandikci** [comment]
@Holmistr Thanks for your contribution 🙂

**Comment by yunussandikci** [comment]
@huseyinbabal I have also another ongoing task for this CLI. I suggest waiting for it for release. Thanks

**Comment by Holmistr** [comment]
I'm done with the changes. Feel free to merge.

**Pull request merged**
